logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/UserEventReceiver.kt

```diff
@@ -31,13 +33,20 @@ class UserEventReceiverImpl(
 
     private suspend fun handleClientRemove(event: Event.User.ClientRemove) {
         clientRepository.currentClientId().map { currentClientId ->
-            if (currentClientId.value == event.id)
+            if (currentClientId == event.clientId)
```

Comment:
We do have the list of self client stored locally, and it needs to be updated in case of not current client + make sure that the BE does not send this event for other users as well

Reply:
where we store "my-clients" list? I can remove it from there as well.

Reply:
i looked up in the code and it's not stored locally my bad